usaspending_api/disaster/v2/views/cfda/spending.py

```diff
@@ -21,20 +22,47 @@ class CfdaSpendingViewSet(ElasticsearchSpendingPaginationMixin, ElasticsearchDis
 
     def build_elasticsearch_result(self, info_buckets: List[dict]) -> List[dict]:
         results = []
+
+        cfda_prefetch_pks = [json_str_to_dict(bucket.get("key")).get("code") for bucket in info_buckets]
+        prefetched_cfdas = Cfda.objects.filter(program_number__in=cfda_prefetch_pks)
+
         for bucket in info_buckets:
             info = json_str_to_dict(bucket.get("key"))
-            results.append(
-                {
-                    "id": int(info.get("id")) if info.get("id") else None,
-                    "code": info.get("code") or None,
-                    "description": info.get("description") or None,
-                    "award_count": int(bucket.get("doc_count", 0)),
-                    "resource_link": info.get("url") or None,
-                    **{
-                        column: get_summed_value_as_float(bucket, self.sum_column_mapping[column])
-                        for column in self.sum_column_mapping
-                    },
-                }
-            )
+
+            toAdd = {
+                "id": int(info.get("id")) if info.get("id") else None,
+                "code": info.get("code") or None,
+                "description": info.get("description") or None,
+                "award_count": int(bucket.get("doc_count", 0)),
+                "resource_link": info.get("url") or None,
+                **{
+                    column: get_summed_value_as_float(bucket, self.sum_column_mapping[column])
+                    for column in self.sum_column_mapping
+                },
+            }
+
+            cfda = [cfda for cfda in prefetched_cfdas if cfda.program_number == info.get("code")]
+            if cfda:
+                toAdd.update(
+                    {
+                        "cfda_federal_agency": cfda[0].federal_agency,
+                        "cfda_objectives": cfda[0].objectives,
+                        "cfda_website": cfda[0].url,
```

Comment:
Can we remove `resource_link` or use that instead? These two fields are the exact same thing.

Reply:
Removing cfda_website sounds like the best idea. @jonhill13 for awareness